meshers/blocky/voxel_library.h

```diff
@@ -86,6 +89,7 @@ class VoxelLibrary : public Resource {
 	std::vector<Ref<Voxel> > _voxel_types;
 	int _atlas_size = 16;
 	bool _needs_baking = true;
+	bool _bake_tangents = false;
```

Comment:
We can make it default to true as it's more an optimization feature, so it won't confuse users who don't know why shading gets wrong with normal maps